src/test/java/com/jitterted/mobreg/adapter/in/web/member/EnsembleSummaryViewTest.java

```diff
@@ -10,108 +13,172 @@
 
         assertThat(ensembleSummaryView.memberStatus())
-                .isEqualTo("full");
+            .isEqualTo("full");
     }
-    
+
     @Test
     public void viewIndicatesCanAcceptIfEnsembleIsNotFull() throws Exception {
         Ensemble ensemble = EnsembleFactory.withIdOf1AndOneDayInTheFuture();
-        MemberFactory.acceptCountMembersWith(ensemble, 2);
+        TestMemberBuilder memberBuilder = new TestMemberBuilder();
+        memberBuilder.createAnsSaveMembers(2);
+        EnsembleFactory.acceptCountMembersFor(2, ensemble);
 
-        EnsembleSummaryView ensembleSummaryView = EnsembleSummaryView.toView(ensemble, MemberId.of(1));
+        EnsembleSummaryView ensembleSummaryView = EnsembleSummaryView.toView(ensemble, MemberId.of(1), memberBuilder.memberService());
 
         assertThat(ensembleSummaryView.memberStatus())
-                .isEqualTo("accepted");
+            .isEqualTo("accepted");
+    }
+
+    @Test
+    public void ensembleWithNoOneAcceptedShowsNoAcceptedMembers() throws Exception {
+        Ensemble ensemble = EnsembleFactory.withIdOf1AndOneDayInTheFuture();
+        MemberService memberService = new MemberService(new InMemoryMemberRepository());
+
+        EnsembleSummaryView ensembleSummaryView = EnsembleSummaryView
+            .toView(ensemble, MemberId.of(11L), memberService);
+
+        assertThat(ensembleSummaryView.acceptedMembers())
+            .isEmpty();
+    }
+
+    @Test
+    public void ensembleWithAcceptedMembersGetsInfoOnAndShowsWhoAccepted() throws Exception {
+        Ensemble ensemble = EnsembleFactory.withIdOf1AndOneDayInTheFuture();
+        TestMemberBuilder memberBuilder = new TestMemberBuilder();
+        MemberService memberService = memberBuilder.memberService();
+        Member member1 = memberBuilder
+                         .withFirstName("one")
+                         .withGithubUsername("github_one")
+                         .buildAndSave();
+        Member member2 = memberBuilder
+                        .withFirstName("two")
+                        .withGithubUsername("github_two")
+                        .buildAndSave();
+        ensemble.acceptedBy(member1.getId());
+        ensemble.acceptedBy(member2.getId());
+
+        EnsembleSummaryView ensembleSummaryView = EnsembleSummaryView
+            .toView(ensemble, MemberId.of(member1.getId().id()), memberService);
+
+        assertThat(ensembleSummaryView.acceptedMembers())
+                        .extracting(MemberView::firstName)
+                        .containsExactlyInAnyOrder("one", "two");
+        assertThat(ensembleSummaryView.acceptedMembers())
+                        .extracting(MemberView::githubUsername)
+                        .containsExactlyInAnyOrder("github_one", "github_two");
     }
 
-}
\ No newline at end of file
+    @Test
+    public void ensembleWithDeclinedMembersShowsNoAcceptedMembers() throws Exception {
```

Comment:
This test kinda feels unnecessary? There are already tests that ensure Declined members don't affect accepted members.